src/telegram/commands/mod.rs

```diff
@@ -43,6 +46,7 @@ async fn handle_command(bot: Bot, command: Command, message: Message) -> Command
 		Command::Fortune => fortune::handler(&bot, &message).await,
 		Command::Echo(text) => echo::handler(&bot, &message, &text).await,
 		Command::WhoAmI => whoami::handler(&bot, &message).await,
+		Command::Answer(_question) => answer::handler(&bot, &message).await,
```

Comment:
Quando vuoi proprio scartare il contenuto di una enum variant puoi usare direttamente `_`!

```suggestion
		Command::Answer(_) => answer::handler(&bot, &message).await,
```